<img src="./unal.png" align=left width="150" height="150"></img>

$\;$
---


<h2 align="center">PRUEBA DE MEDIA (para población normal, $\sigma^2$ desconocida)

    
### Oscar Julian Layton 
---
    
$\;$



Para realizar una prueba de de media considerese una población normal:

## Resultado importante

<div class="alert alert-success">

Si $\{ y_1, y_2 ,..., y_n\}$ es una _muestra aleatoria_ de distribución  $N(\mu,\sigma^2)$, entonces la variable aleatoria:

$$T=\frac{\overline{y}-\mu}{s/\sqrt{n}}   \;\;\;\;\;\;; \;\;\;\;\;\; T \;  \sim  \;  t_{(n-1)}$$

Tengase en cuenta que al decir muestra aleatoria se afirma que la sucecion de variables aleatorias son independientes igualmente distribuidas.
</div>


#### 1. Parámetro Poblacional

El parámetro poblacional a contrastar es $\mu$

#### 2. Hipótesis

Considérese la varianza poblacional $\sigma^2$


\begin{equation}
H_{0}: \mu = \mu_o \hspace{0.9cm} vs \hspace{0.9cm} H_{1}: \mu  > \mu_o
\end{equation}

#### 3. Estadística de prueba

Supóngase que $x_1, x_2, ,..., x_n$ es una muestra aleatoria de **X**, tal que $X \sim N(\mu, \sigma^2)$, entonces:

\begin{equation}
T = \frac{\overline{X}- \mu}{s /\sqrt{n}}\hspace{0.4cm} \hspace{1.2cm} ;  \hspace{1.2cm} T \mid _{H_0} \sim  t_{(n-1)}
\end{equation}

#### 4. Construcción regla de desición

Para un $0<\alpha<1$ nivel de significancia

$$P_{\mu_{o}}(Error\; tipo \; I)=\alpha$$
$$P_{\mu_{o}}(Rechaz\;H_o\;cuando \;es \;verdadera)=\alpha$$
$$P_{\mu_{o}}(T \geq  t_{\alpha(n-1)} \; cuando \; T \sim  t_{(n-1)})=\alpha$$

En consecuencia:

$$P_{\mu_{o}}\left (   \frac{\overline{X}_n- \mu}{s/\sqrt{n}}\geq  t_{\alpha(n-1)} \right )=\alpha$$

#### 5. Regla de Desición


<div class="alert alert-danger">

Para un $\alpha$ de significancia se procede a:

\begin{equation}
\vartheta :  \; \;Rechazar \; H_o  \;\;\;\; sii \;\; \;\; T \geq  t_{\alpha(n-1)}
\end{equation}
</div>

---


Para la **potencia de la prueba** se usa el error tipo II

$$\beta(\mu')=P(Error\; tipo \; II) $$

$$\beta(\mu')= P(T <  t_{\alpha(n-1)} \; cuando \; \mu =\mu^{´})$$


$$\beta(\mu') = P(\overline{X}_n < \mu_o +  t_{\alpha(n-1)}(s/\sqrt{n}) \;\; con \;\; \mu =\mu^{´})$$

$$\beta(\mu') =  \left (   \frac{\overline{X}_n- \mu^{´}}{s/\sqrt{n}} <  t_{\alpha(n-1)} + \frac{\mu_o- \mu^{´}}{s/\sqrt{n} } \right )$$
$$potencia = 1 - \beta$$

---
## Ejemplo

Se seleccionaron al azar sujetos de una población de varones saludables de 4-7 años de edad, considerando que el peso máximo está normalmente distribuido. ¿Sugieren los siguientes datos afirmar que el peso máximo de la población excede de 25kg?  

$$25.8,\;\; 36.6,\;\; 26.3,\;\; 21.8,\;\; 27.2$$

#### a) Hipótesis apropiadas sobre la media.

En este caso interesa hacer inferencia con respecto a $\mu$ peso máximo promedio de la población. En este orden de ideas la hipótesis a contrastar es:

\begin{equation}
H_{0}: \mu = 25 \hspace{0.9cm} vs \hspace{0.9cm} H_{1}: \mu > 25
\end{equation}

#### b) Uso de la estadística de prueba.

Tenemos entonces los siguientes datos: $n=5$, $\bar{x}=0.2545$, se usa el estadístico de prueba: 

$$T=\frac{\overline{x}-\mu}{s/\sqrt{n}}   \;\;\;\;\;\;; \;\;\;\;\;\; T \;  \sim  \;  t_{(n-1)}$$


In [2]:
data <- c(25.8, 36.6, 26.3, 21.8, 27.2)

In [15]:
mu<-25
T <- (mean(data)-mu)*sqrt(length(data))/sd(data)  ; T

[1] 1.038197

De esta forma, para la información disponible tenemos que: 

$$T=\frac{\sqrt[]{n}(\bar{x}-\mu_{0})}{s}=\frac{\sqrt[]{5}(27.54-25)}{5.47}=1.038$$

Luego se tiene:

In [16]:
##########################  cuantil distribucion t ################################################################
######     distribución t     qt(p, df, ncp, lower.tail = TRUE, log.p = FALSE)  #si es central npc=0  #############  
###################################################################################################################

alpha=0.05
df=length(data)-1    #por ser dos muestras 

qt(alpha,df, 0, lower.tail = FALSE)   #alpha/2 para prueba dos colas,    # lower.tail=TRUE ent P[X <= x]

[1] 2.131847

$$t_{\alpha(n-1)} = t_{0.05(5-1)} =2.13$$

#### c) Desición

 $ \vartheta :$ Luego como $T= 0.102$, se tiene que a un nivel de significancia de $ \alpha= 0.05 $, $\;\;$ $T= 1.038 < t_{[0.05\;,\;4]}=2.13 $ con lo cual se concluye que NO hay evidencia estadísticamente significativa para rechazar la hipotesis nula, luego no puede aceptarse la afirmación de que el peso de los varones de 4-7 años execda los 25kg (cuanto mucho será 25kg). esto tambien se evidencia en el p_valor (el cual es muy grande):

In [10]:
pt(T,df, 0, lower.tail = FALSE)   #alpha/2 para prueba dos colas,    # lower.tail=TRUE ent P[X <= x]

[1] 0.1789067

---
#### d) Prueba en R junto con intervalo de confianza

In [13]:
t.test(x = data,  alternative = "greater",  mu = 25, conf.level  = 0.95)  
#alternative="greater" or "less" or two.sided


	One Sample t-test

data:  data
t = 1.0382, df = 4, p-value = 0.1789
alternative hypothesis: true mean is greater than 25
95 percent confidence interval:
 22.32433      Inf
sample estimates:
mean of x 
    27.54 
